test/Api.Test/Controllers/CollectionsControllerTests.cs

```diff
@@ -85,4 +86,182 @@ await sutProvider.GetDependency<ICollectionService>()
 
         _ = await Assert.ThrowsAsync<NotFoundException>(async () => await sutProvider.Sut.Put(orgId, collectionId, collectionRequest));
     }
+
+    [Theory, BitAutoData]
+    public async Task GetOrganizationCollectionsWithGroups_NoManagerPermissions_ThrowsNotFound(Organization organization, SutProvider<CollectionsController> sutProvider)
```

Comment:
Love that the tests are testing each of the user type/permission combinations!

Reply:
😄